mtags/src/main/scala/scala/meta/internal/mtags/SymbolIndexBucket.scala

```diff
@@ -36,7 +37,7 @@ final case class SymbolLocation(
 class SymbolIndexBucket(
     toplevels: TrieMap[String, Set[AbsolutePath]],
     definitions: TrieMap[String, Set[SymbolLocation]],
-    sourceJars: ClasspathLoader,
+    sourceJars: OpenClassLoader,
```

Comment:
Could we remove `ClasspathLoader` now actually? `ClasspathLoader` is really just used in tests and I think we could just read the classpath java property instead int hose cases.

Reply:
I'll try to remove it.

Reply:
Thanks!